db/seeds/evaluatieonderzoek/questionnaires/evaluatieonderzoek.rb

```diff
@@ -75,7 +81,7 @@
                        type: :dropdown,
                        label: 'RMC-regio',
                        title: 'Bij welke RMC-regio hoort uw gemeente?',
-                       tooltip: '<p>Als u het niet zeker weet, kunt u op dit kaartje kijken:</p><img width="90%" src="/evaluatieonderzoek/rmcregios.png">',
+                       tooltip: '<p>Als u het niet zeker weet, kunt u op dit kaartje kijken:</p><img height="640" src="/evaluatieonderzoek/rmcregios.png">',
```

Comment:
same here en overal waar je dit had aangepast

Reply:
Fixed.

Ik het het laten staan als het maar 1 keer voorkomt, maar als het vaker voorkomt kan het idd in een variabele.